### PR TITLE
feat: Vietnamese Localizations

### PR DESCRIPTION
**Pull request by ohaiibuzzle**
I couldn't submit via Weblate as, at the time of this PR anyway, it has a merge conflict.
Added translations for Vietnamese. 

image

### CODE REVIEW

**Comment by Blinue** [comment]
Thank you for your contribution! Please merge into the "dev" branch. By the way, the merge conflicts in Weblate have been resolved. It appears that this error occurs when there are other commits before merging a Weblate PR.

File: src/Magpie.App/Magpie.App.vcxproj.filters [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="4.0"
+  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (Blinue) [inline src/Magpie.App/Magpie.App.vcxproj.filters:3]: unrelated?

File: src/Magpie.Core/OverlayDrawer.cpp [hunk 2]
```diff
@@ -357,6 +357,8 @@ void OverlayDrawer::_BuildFontUI(std::wstring_view language, const std::vector<u
 		builder.AddRanges(fontAtlas.GetGlyphRangesCyrillic());
 	} else if (language == L"tr" || language == L"hu") {
 		builder.AddRanges(ImGuiHelper::Latin_1_Extended_A_RANGES);
+	} else if (language == L"vi") {
+		builder.AddRanges(fontAtlas.GetGlyphRangesVietnamese());
```
Comment (Blinue) [inline src/Magpie.Core/OverlayDrawer.cpp:361]: This surprised me; I had intended to fix this in another commit. Great job!

Comment (ohaiibuzzle) [reply src/Magpie.Core/OverlayDrawer.cpp:361]: Yeah, caught it while checking everything, might as well fix it

**Comment by Blinue** [comment]
LGTM! These translations will be automatically synchronized to Weblate, so please make future updates there.

**Comment by Blinue** [comment]
@all-contributors
Please add @ohaiibuzzle for translation.
